src/trackers/strategy/strategyOptimized.ts

```diff
@@ -25,7 +25,7 @@ export function strategyOptimizedFactory(
         const now = Date.now();
         
         // Increments impression counter per featureName
-        impressionsCounter.track(impression.feature, now, 1);
+        if (impression.pt !== 0) impressionsCounter.track(impression.feature, now, 1);
```

Comment:
Emma, replace with `if (impression.pt)` to handle the case where pt can be undefined, and should not be tracked